src/sentry/snuba/metrics/fields/base.py

```diff
@@ -1785,21 +1785,21 @@ def generate_where_statements(
         ),
         # Custom operations used for on demand derived metrics.
         DerivedOp(
-            op="on_demand_failure_count",
+            op="on_demand_apdex",
```

Comment:
I'm just sorting these in alphabetical order.

Ideally, we would define all these `DerivedOp` by iterating over `DERIVED_METRICS`.